Components/Core/Inc/Command.hpp

```diff
@@ -19,7 +19,8 @@ enum GLOBAL_COMMANDS : uint8_t
 {
 	COMMAND_NONE = 0,		// No command, packet can probably be ignored
 	TASK_SPECIFIC_COMMAND,	// Runs a task specific command when given this object
-	DATA_COMMAND			// Data command, used to send data to a task. Target is stored in taskCommand
+	DATA_COMMAND,			// Data command, used to send data to a task. Target is stored in taskCommand
+	REQUEST_COMMAND			// Request command
 };
 
 /* Class -----------------------------------------------------------------*/
```

Comment:
Do we want to explicitly delete the copy cstor and assignment optor for commands?

Reply:
Made copy constructor private: 31c46ef354daa3357a5cd605480e769e832bf5b8